manifests/20-performance-profile.crd.yaml

```diff
@@ -382,6 +382,15 @@ spec:
                     reserved:
                       description: Reserved defines a set of CPUs that will not be used for any container workloads initiated by kubelet.
                       type: string
+                    workloads:
+                      description: Define workload partitioning configurations. The specified workloads annotation will be pinned to the defined "reserved" CPUs
+                      type: array
+                      items:
+                        description: WorkloadAnnotation defines which workload annotation to pin
```

Comment:
Should you maybe document that only one value is supported atm?

Reply:
Resolving: Changed approach no longer includes PA API Change